DietPi-Software | Home Assistant: Install HACS by default

Joulinar opened the pull request:
DietPi-Software | Home Assistant - add HACS plugin by default to HA install

Joulinar committed:
v8.10

DietPi-Software | Home Assistant - add HACS plugin by default to HA install

MichaIng commented on dietpi/dietpi-software:
```diff
@@ -12119,7 +12119,13 @@ _EOF_
+
+			# Download HACS
+			Download_Install 'https://github.com/hacs/integration/releases/latest/download/hacs.zip' hacs
+			[[ -d '/mnt/dietpi_userdata/homeassistant/custom_components/hacs' ]] && G_EXEC rm -Rf /mnt/dietpi_userdata/homeassistant/custom_components/hacs
```
`-f` flag means to ignore when file/dir is missing, but we explicitly check whether it exists:
```suggestion
			[[ -d '/mnt/dietpi_userdata/homeassistant/custom_components/hacs' ]] && G_EXEC rm -R /mnt/dietpi_userdata/homeassistant/custom_components/hacs
```

MichaIng committed:
v8.10

- Home Assistent | Update Python to v3.9.15 and minor coding/order enhancements, e.g. do not remove old pyenv before dependencies have been installed and new pyenv successfully downloaded

MichaIng committed:
v8.10

- Home Assistant | Add/align HACS integration info

MichaIng committed:
v8.10

- Home Assistant | Fix HACS install
